Document dry run flag (#99)
kyma-project/compass-manager: README.md

```
## Usage

### Configuration Envs

| Name                               | Default                                                                      | Description                                                                        |
|------------------------------------|------------------------------------------------------------------------------|------------------------------------------------------------------------------------|

> | `APP_ADDRESS`                      | `127.0.0.1:3000`                                                             | Adress on which the app will be exposed                                            |

Reviewer: ```suggestion
| `APP_ADDRESS`                      | `127.0.0.1:3000`                                                             | Address on which the app is exposed                                            |
```

| `APP_APIENDPOINT`                  | `/graphql`                                                                   | Endpoint for GraphQL requests                                                      |

> | `APP_PLAYGROUNDAPIENDPOINT`        | `/graphql`                                                                   | This option is not used                                                            |

Reviewer: If this option is not used, do we need it here?

> | `APP_SKIPDIRECTORCERTVERIFICATION` | `false`                                                                      | Skip cert verification in Compass Director GraphQL calls                           |

Reviewer: ```suggestion
| `APP_SKIPDIRECTORCERTVERIFICATION` | `false`                                                                      | Skips cert verification in the Compass Director GraphQL calls                           |
```

| `APP_DIRECTOR_URL`                 | `https://compass-gateway-auth-oauth.mps.dev.kyma.cloud.sap/director/graphql` | URL of the Compass Director GraphQL endpoint                                       |
| `APP_DIRECTOR_OAUTH_PATH`          | `./dev/director.yaml`                                                        | File with OAuth data for Compass Director                                          |
| `APP_ENABLED_REGISTRATION`         | `false`                                                                      | Enable registering runtimes with Compass                                           |

> | `APP_DRYRUN`                       | `false`                                                                      | Disable registering and configuring, instead log what operations would be executed |

Reviewer: ```suggestion
| `APP_DRYRUN`                       | `false`                                                                      | Disable registering and configuring; instead log which operations would be executed |
```

> > **TIP:** `CompassManagerMappings` created with dry run will be labeled `kyma-project.io/cm-dry-run: Yes`

Reviewer: ```suggestion
> **TIP:** `CompassManagerMappings` created with dry run are labeled `kyma-project.io/cm-dry-run: Yes`
```
